add html (header/hero) fix (layout\module)

KuryliakPetro opened the pull request.

Commit:
add html (header/hero) fix (layout\module)

FinikBranson commented on src/partials/header.html:
```diff
@@ -0,0 +1,23 @@
+<section class="header">
+  <a href="#"><img src="../image/Vector-7.svg" alt="logo" /></a>
```
Картинка в проекте отсутствует, добавь картинку или убери путь.

FinikBranson replied:
Картинка в проекте отсутствует, добавить картинку или убрать путь, сборка не видит искомого

FinikBranson commented on src/partials/header.html:
```diff
@@ -0,0 +1,23 @@
+    <li>
+      <a href="#about">About us</a>
+    </li>
+    <li>
```
енрен

FinikBranson commented on src/partials/hero.html:
```diff
@@ -0,0 +1,21 @@
+<section class="hero" id="hero">
+  <h1>ice cream made with passion</h1>
```
последнее слово в заголовке обернуть в span

FinikBranson reviewed (commented):
У меня в архитектуре было два саса с именем header, это очепятка, вообще должно было быть header and hero. 2 файла. Если тебе не удобно так, работай в одном, но думаю лучше в двух разных

FinikBranson commented on src/partials/header.html:
```diff
@@ -0,0 +1,23 @@
+<section class="header">
+  <a href="#"><img src="../image/Vector-7.svg" alt="logo" /></a>
+
```
Думаю еще нужно добавить в разметку кнопку открытия модального окна (меню). Все что ниже обернуть в div и тоже добавить кнопку закрытия. Ссылки по навигации сайта обернуть в nav

Commit:
fixx hero/header

FinikBranson reviewed (commented):
Поправь разметку

Commit:
fix

добавил в модули hero, добавил span (не понимаю какое єто имеет значение для html?)  кнопки модального меню добавлю когда их буду делать бургер меню
